manifests/0000_08_cluster-dns-operator_03-daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+# Node resolver for internal registry
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cluster-dns-node-resolver
+  namespace: openshift-cluster-dns-operator
+  labels:
+    openshift-app: dns-node-resolver
+spec:
+  selector:
+    matchLabels:
+      openshift-app: dns-node-resolver
+  template:
+    metadata:
+      labels:
+        openshift-app: dns-node-resolver
+    spec:
+      dnsPolicy: Default
+      tolerations:
+      # tolerate all taints so that DNS node resolver is always present on all nodes
+      - operator: Exists
+      serviceAccountName: default
+      volumes:
+       - name: hosts-file
+         hostPath:
+           path: /etc/hosts
+           type: File
+      containers:
+      - name: dns-node-resolver
+        image: "openshift/origin-cli:v4.0"
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: hosts-file
+          mountPath: /etc/hosts
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -euo pipefail
+
+          # TODO: Is this the correct service name for internal registry?
+          # Is there one global registry or multiple registries for dev, test and external users?
+          REGISTRY_SERVICE="docker-registry.default.svc.cluster.local"
```

Comment:
Omit the `cluster.local` since that's configurable and should be in the container resolver search path.